pubtools/_quay/operator_pusher.py

```diff
@@ -623,7 +701,8 @@ def build_index_images(self):
                     "iib_result": build_details,
                     "signing_keys": param.signing_keys,
                     "is_hotfix": param.is_hotfix,
-                    "hotfix_tag": param.hotfix_tag,
+                    "destination_tags": param.destination_tags,
+                    # "hotfix_tag": param.hotfix_tag,
```

Comment:
Remove this? signature_handler.py also needs update to sign hotfix and prerelease images.